HelpMyStreetFE/HelpMyStreetFE/Views/Shared/DisplayTemplates/RequestSummaryViewModel.cshtml

```diff
@@ -119,8 +137,8 @@
                         </span>
                     </span>
 
-                    var references = Model.Item.JobSummaries.Where(j => j.SupportActivity.Equals(activity) && !string.IsNullOrWhiteSpace(j.Reference)).Select(j => j.Reference).Distinct();
-                    if (references.Count() > 0)
+                    var references = Model.Item.JobSummaries.Where(j => j.SupportActivity.Equals(activity)).Select(x => x.Reference).Distinct();
+                    if (references.Count() > 0 && references.First() != null)
```

Comment:
Haven't tested this, but is there a risk that we end up with references being a list, where the first item is null and the second is non-null?  If so, we'd end up not displaying the second one.